src/lib/ewelink.ts

```diff
@@ -31,6 +722,108 @@ export const ewelinkModernExtend = {
             voltageReportingConfig: {min: 3600, max: 7200, change: 10},
         });
     },
+    ewelinkWindowCovering(args: WindowCoveringArgs): ModernExtend {
+        args = {stateSource: 'lift', configureReporting: true, ...args};
```

Comment:
Could you call the `windowCovering` modernExtend here and extend based on that instead of duplicating all the code?